up-l3/utwin/v2/README.adoc

```diff
@@ -111,6 +111,16 @@ A Topic is a cache entry that can be refreshed individually. Refreshing cache in
 
 uTwin instances act as a distributed networks of caches that can communicate to retrieve values without requesting a new publication, in case a uTwin instance is lacking a Topic entry in its own cache, by retrieving the last event from the cache (uTwin instance) running on the same device as the publisher of the topic). Using the analogy of a cache, it may also be useful to implement, later, a flush() command, that would remove the last known state for specific Topics, or all Topics of a given uDevice, etc.
 
+=== uTwin Cache Management
+
+uTwin will initialize its cache on startup with the following scenarios:
+
+* when running without persistent storage, the cache will be initialized to an empty condition
+* when running with persistent storage (future option), the cache can be initialized to the last saved state for each Topic.
+
+uTwin MUST indicate when its capacity is exhausted and it is unable to accept a new / updated SetLastValue command.
```

Comment:
MUST needs to be bolded and I don't follow the requirement. We only cache the last known event per-topic so is this requirement for when we can no longer cache any topics as the cache is full?